src/main/java/de/konfidas/ttc/messages/LogMessage.java

```diff
@@ -120,6 +121,36 @@ public String getSignatureAlgorithm(){
 
     public BigInteger getSignatureCounter(){ return signatureCounter; }
 
+    private byte[] getLFromASN1(ASN1Primitive element) throws IOException{
+        byte[] elementContent = element.getEncoded();
+
+        if (((int)elementContent[0])<=128){
```

Comment:
< 128. 

see section 3.1 of http://luca.ntop.org/Teaching/Appunti/asn1.html
(a length of 128 is encoded as {0x81, 0x80})

in case of = 128 = 0x80 we have indefinite length encoding.